src/main/java/edu/iis/powp/command/ComplexCommand.java

```diff
@@ -1,21 +1,20 @@
 package edu.iis.powp.command;
 
 import java.util.ArrayList;
-
 import edu.iis.client.plottermagic.IPlotter;
 
 public class ComplexCommand implements PlotterCommand {
+
+	private ArrayList<PlotterCommand> commandList = new ArrayList<>();
```

Comment:
Variable type should be List (indication of an expected interface). ArrayList is an implementation.